content/docs/rfcs/14/README.md

```diff
@@ -69,6 +69,13 @@ This provides for asymmetric and symmetric encryption.
 Key agreement is out of band.
 It also provides an encrypted signature and padding for some form of unlinkability.
 
+### Version 2
+
+This indicates that payloads MUST be encoded using [35/WAKU-NOISE envelope data format spec](/spec/35).
```

Comment:
```suggestion
This indicates that payloads MUST be encoded using [35/WAKU-NOISE](/spec/35).
```

Reply:
Addressed in https://github.com/vacp2p/rfc/pull/496/commits/8becd7bf7c5bf8575877b8cdc4444a4b9e5d1463